pkg/config/defaults.go

```diff
@@ -13,7 +13,6 @@ func newDefaults(project Project) Config {
 		Branches: Branches{
 			Main:        "main",
 			ReleaseNext: "release-next",
-			SynchCI:     "ci/",
```

Comment:
Oh. I missed this. I need to restore this line, as it breaks daily check PR on kn-event.

Reply:
Or maybe better to introduce a skip flag. Configuring an empty value might not be possible, due to processing of the defaults.

Reply:
https://github.com/openshift-knative/deviate/pull/48 fixes this